projects/sbb-esta/angular-business/src/lib/button/button/button.component.ts

```diff
@@ -0,0 +1,35 @@
+import {
+  ChangeDetectionStrategy,
+  Component,
+  HostBinding,
+  Input,
+  ViewEncapsulation
+} from '@angular/core';
+import { BaseButton } from '../../../../../angular-public/src/lib/button/button/base-button';
+
+@Component({
+  // tslint:disable-next-line:component-selector
+  selector: 'button[sbbButton], input[type=submit][sbbButton]',
+  templateUrl: '../../../../../angular-public/src/lib/button/button/button.component.html',
+  styleUrls: ['../../../../../angular-public/src/lib/button/button/button.component.scss'],
```

Comment:
I'm wondering, if it would make more sense to separate styling a bit more, since there are more differences between public and business styles for buttons. (Icon animation, disabled styling, different button types, ...)
What is your opinion?

Reply:
I split up the button types to seperate files. For now, I left the arrowbuttons and animations as is. I have to check if this is compliant to the style guide next week.